docs/dynamic_rules.md

```diff
@@ -492,7 +492,9 @@ eval(${"var"});
 
 ##### `@filter`
 
-The `@filter` restriction allows you to restrict the rule by name of matched variable.
+The `@filter` restriction allows you to restrict the rule by name of matched variable or literal, const or anything else! 
+
+More about which can be captured you can read in [phpgrep doc](https://github.com/quasilyte/phpgrep/blob/master/docs/user_manual.md#-filter)
```

Comment:
why not https://github.com/VKCOM/noverify/blob/master/src/phpgrep/README.md?

Reply:
Because throw my link you can see the table with supported types

Reply:
Wrong link, wrong link: https://github.com/VKCOM/noverify/blob/master/src/phpgrep/pattern_language.md#matcher-expressions